src/Chapter4.hs

```diff
@@ -611,7 +630,15 @@ Implement the 'Monad' instance for our lists.
   maybe a few) to flatten lists of lists to a single list.
 -}
 
+instance Monad List where 
+  (>>=) :: List a -> (a -> List b) -> List b
+  Empty >>= _ = Empty
+  Cons x xs >>= f = append (f x) (xs >>= f)
 
+replicate' :: Int -> a -> List a
+replicate' n x | n == 1 = Cons x Empty
+               | n > 1 = Cons x (replicate' (n-1) x)
+               | otherwise = Empty
```

Comment:
That's a nice function! I think you can improve the implementation a little bit. If you check just two conditions: `n < 1` and `otherwise`, then you will be able to write this a little bit shorter.